# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../perl/lib64.diff
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

Mostly imported from Fedora Core ,-)

  - Rene Rebe <rene@exactcode.de>

--- perl-5.8.0/Configure.orig	2002-09-09 11:31:19.000000000 -0400
+++ perl-5.8.0/Configure	2002-09-09 11:40:37.000000000 -0400
@@ -6458,8 +6458,8 @@
 : Reproduce behavior of 5.005 and earlier, maybe drop that in 5.7.
 case "$installstyle" in
 '')	case "$prefix" in
-		*perl*) dflt='lib';;
-		*) dflt='lib/perl5' ;;
+		*perl*) dflt='lib64';;
+		*) dflt='lib64/perl5' ;;
 	esac
 	;;
 *)	dflt="$installstyle" ;;
@@ -6475,8 +6475,8 @@
 : /opt/perl/lib/perl5... would be redundant.
 : The default "style" setting is made in installstyle.U
 case "$installstyle" in
-*lib/perl5*) set dflt privlib lib/$package/$version ;;
-*)	 set dflt privlib lib/$version ;;
+*lib64/perl5*) set dflt privlib lib64/$package/$version ;;
+*)	 set dflt privlib lib64/$version ;;
 esac
 eval $prefixit
 $cat <<EOM
@@ -6934,8 +6934,8 @@
 prog=`echo $package | $sed 's/-*[0-9.]*$//'`
 case "$sitelib" in
 '') case "$installstyle" in
-	*lib/perl5*) dflt=$siteprefix/lib/$package/site_$prog/$version ;;
-	*)	 dflt=$siteprefix/lib/site_$prog/$version ;;
+	*lib64/perl5*) dflt=$siteprefix/lib64/$package/site_$prog/$version ;;
+	*)	 dflt=$siteprefix/lib64/site_$prog/$version ;;
 	esac
 	;;
 *)	dflt="$sitelib"
@@ -7061,8 +7061,8 @@
 	'')
 		prog=`echo $package | $sed 's/-*[0-9.]*$//'`
 		case "$installstyle" in
-		*lib/perl5*) dflt=$vendorprefix/lib/$package/vendor_$prog/$version ;;
-		*)	     dflt=$vendorprefix/lib/vendor_$prog/$version ;;
+		*lib64/perl5*) dflt=$vendorprefix/lib64/$package/vendor_$prog/$version ;;
+		*)	     dflt=$vendorprefix/lib64/vendor_$prog/$version ;;
 		esac
 		;;
 	*)	dflt="$vendorlib"
